src/codeu/chat/client/core/View.java

```diff
@@ -69,6 +70,27 @@ public View(ConnectionSource source) {
   }
 
   @Override
```

Comment:
After reading my later comments - remove `@Override` since this method shouldn't be in the BasicView interface